module/VuFind/src/VuFind/Search/Solr/Params.php

```diff
@@ -489,6 +489,18 @@ protected function normalizeSort($sort)
                 );
             }
         }
+        $tieBreaker = $this->getOptions()->getSortTieBreaker();
+
+        if (!empty($tieBreaker)
+            && !((in_array($tieBreaker . " asc", $normalized)
```

Comment:
I think there's an extra set of parentheses here -- we seem to have `((..))` around the two in_array checks when `(..)` would suffice.